ArduPlane/ArduPlane.cpp

```diff
@@ -79,6 +79,24 @@ const AP_Scheduler::Task Plane::scheduler_tasks[] = {
     SCHED_TASK(update_logging1,        25,    300),
     SCHED_TASK(update_logging2,        25,    300),
     SCHED_TASK(update_soaring,         50,    400),
+    /* Break up the soaring POMDP calculation into
+    several small time slices to avoid starving
+    other vital processes. */
```

Comment:
The method is going to be simplified for now to avoid the need for these calcs until a threaded version is developed. See discussion https://github.com/Microsoft/Frigatebird/issues/5

Reply:
woah, this is a bit crazy. We could do a new thread like @tridge suggests or simply run thus thread faster and state-machine it